src/code_80005FD0.c

```diff
@@ -908,75 +916,92 @@ void func_800074D4(void) {
     }
 }
 
-s32 func_80007BF8(u16 arg0, u16 arg1, u16 arg2, u16 arg3, u16 arg4) {
+/**
+ * Checks if a waypoint is within a valid range of another waypoint, accounting for track wrapping
+ *
+ * @param waypoint The waypoint to check
+ * @param currentWaypoint The reference waypoint
+ * @param backwardRange Number of waypoints to check behind
```

Comment:
to look behind and to look ahead?

(just seems like more common terminology for this sort of concept in programming)